debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java

```diff
@@ -343,6 +343,15 @@
                     "bigger than log.mining.scn.gap.detection.gap.size.min, and the time difference of current SCN and previous end SCN is smaller than " +
                     " this value, consider it a SCN gap.");
 
+    public static final Field TOASTED_VALUE_PLACEHOLDER = Field.create("toasted.value.placeholder")
```

Comment:
So as you say, we definitely should not talk about this as "TOAST". This term is specific to Postgres, and its usage will be confusing here. E.g. "unavailable.value.placeholder" could work. While at it, we also should

* Deprecate the existing option in the Postgres connector and add a new one with the new name
* Add an item to the list in https://issues.redhat.com/browse/DBZ-3899 (Debezium 2.0) for removing the old one